lib/api/repository/login_repository.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter_survey/api/request/login_request.dart';
+import 'package:flutter_survey/api/service/login_service.dart';
+import 'package:flutter_survey/model/login_model.dart';
+import 'package:flutter_survey/api/exception/network_exceptions.dart';
+import 'package:flutter_survey/usecases/login_use_case.dart';
+import 'package:injectable/injectable.dart';
+import 'package:flutter_survey/env.dart';
+
+abstract class LoginRepository {
```

Comment:
This repository will not be used for the login action only, right? Can we rename it to a common name, such as AuthenticationRepository?